testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/rule/AbstractKeycloakRule.java

```diff
@@ -0,0 +1,399 @@
+/*
```

Comment:
I think this file shouldn't be added to the new testsuite. And it seems it is not needed for any model test?